plugins/linux-capture/pipewire.c

```diff
@@ -116,6 +126,32 @@ struct dbus_call_data {
 
 /* auxiliary methods */
 
+static bool parse_pw_version(struct obs_pw_version *dst, const char *version)
+{
+	int n_matches = sscanf(version, "%d.%d.%d", &dst->major, &dst->minor,
+			       &dst->micro);
+	return n_matches == 3;
+}
+
+static bool check_pw_version(const struct obs_pw_version *pw_version, int major,
+			     int minor, int micro)
+{
+	return pw_version->major >= major && pw_version->minor >= minor &&
+	       pw_version->micro >= micro;
```

Comment:
Now that I'm looking... I don`t think this will work work for e.g. `1.1.1`. I think this should work:

```c
if (pw_version->major != major)
	return pw_version->major > major;

if (pw_version->minor != minor)
	return pw_version->minor > minor;

return pw_version->micro > micro;
```

Reply:
whoopsi. Thanks for catching